src/app/Shared/Services/Settings.service.tsx

```diff
@@ -87,6 +95,28 @@ export class SettingsService {
     window.localStorage.setItem(StorageKeys.AutoRefreshUnits, String(units));
   }
 
+  automatedAnalysisRecordingConfig(
+    defaultConfig = defaultAutomatedAnalysisRecordingConfig
+  ): AutomatedAnalysisRecordingConfig {
+    const raw = window.localStorage.getItem(StorageKeys.AutomatedAnalysisRecordingConfig);
```

Comment:
We should use the `getFromLocalStorage` from `@app/utils/LocalStorage` for consistency?